src/Domain/Requests/RegisterAccountPayout.cs

```diff
@@ -0,0 +1,200 @@
+
+        /// <summary>
+        /// Last name of the account holder (empty for organizations/companies)
+        /// </summary>
+        public string Lastname { get; set; }
+
+        /// <summary>
+        /// The URL to which notifications for this payment should be sent to. This URL should be hard to guess and not contain a ? ("question mark").
+        /// </summary>
+        [Required]
+        public string NotificationURL { get; set; }
+
+        /// <summary>
+        /// Your unique ID for the payout.
+        /// If the MessageID is a previously initiated P2P order then the payout will be attached to that P2P order and the amount must be equal to or lower than the previously deposited amount.
+        /// </summary>
+        [Required]
+        public string MessageID { get; set; }
+
+        /// <summary>
+        /// The amount to send with exactly two decimals. Only digits. Use dot (.) as decimal separator.
+        /// If the end-user holds a balance in the merchant's system then the amount must have been deducted from that balance before calling this method.
+        /// </summary>
+        [Required]
+        public string Amount { get; set; }
+
+        /// <summary>
+        /// The currency of the end-user's account in the merchant's system.
+        /// </summary>
+        [Required]
+        public string Currency { get; set; }
+    }
+
+	public class RegisterAccountPayoutRequestDataAttributes : AbstractRequestParamsDataAttributes
+	{
+        /// <summary>
+        /// The text to show on the end-user's bank statement after Trustly's own 10 digit reference (which always will be displayed first).
+        /// The reference must let the end user identify the merchant based on this value. So the ShopperStatement should contain either your brand name, website name, or company name.
+        /// 
+        /// If possible, try to keep this text as short as possible to maximise the chance that the full reference
+        /// will fit into the reference field on the customer's bank since some banks allow only a limited number of characters.
+        /// </summary>
+        [Required]
+        public string ShopperStatement { get; set; }
+
+        /// <summary>
+        /// The ExternalReference is a reference set by the merchant for any purpose and does not need to be unique
+        /// for every API call. The ExternalReference will be included in version 1.2 of the settlement report, ViewAutomaticSettlementDetailsCSV.
+        /// </summary>
+        public string ExternalReference { get; set; }
+
+        /// <summary>
+        /// Human-readable identifier of the consumer-facing merchant (e.g. legal name or trade name)
+        /// <remarks>Mandatory attribute for Trustly Partners that are using Express Merchant Onboarding (EMO) and aggregate traffic under a master processing account. It is also mandatory for E-wallets used directly in a merchant's checkout.</remarks>
+        /// </summary>
```

Comment:
I think `remarks` is supposed to be outside/next to the `summary`, like with `example`